docs/docs/guide/debugging.mdx

```diff
@@ -0,0 +1,200 @@
+functions. This means that the `scrollTo` function will work
+(using the native web implementation), but the `measure`
+function will not be available and it's usage will trigger this error:
+
+```
+[reanimated.measure] method cannot be used for web or Chrome Debugger
+```
+
+You may stil use the standard web version of measure as described
+[here](../api/nativeMethods/measure).
+
+Those functions that are provided by Reanimated and do not have web
+implementations won't work at all.
+An example of this behaviour is the `useAnimatedSensor` hook
+which currently only works on mobile platforms. When debugging in Chrome and using
+this hook the following message will appear in the logs:
+
+```
+[Reanimated] useAnimatedSensor is not available on web yet.
+```
+
+But despite all of this, it is still possible to set breakpoints both in
+normal JS code as well as in worklets (since they run on the main JS
+thread now).
+
+### Chrome DevTools
+
+<ChromeDevToolsTable />
+
+:::info
+
+Works, but only the JS context can be debugged.
+
+:::
+
+Even though Chrome DevTools supports the Hermes and V8 engines it
+unfortunately doesn't recognize Reanimated's additional UI context. This
+means that you won't be able to debug worklets and breakpoints set in them
+will be ignored. All other features work as expected.
+
+*We are actively working on enabling worklet debugging with Chrome DevTools on Hermes.*
+
+### Flipper (Hermes debugger)
+
+<FlipperTable />
+
+:::info
+
+Works, but only the JS context can be debugged.
+
+:::
+
+Even though Flipper supports the Hermes and V8 engines it
+unfortunately doesn't recognize Reanimated's additional UI context. This
+means that you won't be able to debug worklets and breakpoints set in them
+will be ignored. All other features work as expected.
+
+*We are actively working on enabling worklet debugging with Flipper on Hermes.*
+
+### Safari DevTools
+
+<SafariDevToolsTable />
+
+:::info
+
+Safari DevTools is only available on iOS devices running the JSC engine.
+
+:::
+
+After opening Safari and configuring it as specified in the React Native
+docs, under `Develop > Device` you'll see two JSC contexts
+like in the screenshot below:
+
+![Screenshot showing Safari's Develop menu options](/img/debugging/SafariJSCiOS.png)
+
+One of them will be the main JS context. All `console.log`
+outputs will appear in the console of this context. You can also set
+breakpoints here, but unfortunately the only source file visible is the
+transformed `indexjs.bundle` which does make things more
+difficult to find.
+
+The other option will be the UI context. No console logs will appear in
+the console of this context, but all worklet functions should be visible
+as separate files. It is also possible to set breakpoints in these
+worklets.
+
+**Caution!**
+*Remember that console logs will appear on the main thread as the
+`console.log` funcion on the UI thread is just a reference to the one from
+the JS thread.*
```

Comment:
This applies to all debugging tools so we can move this higher. Also, use `:::caution` to show a yellow box.